babel/messages/extract.py

```diff
@@ -29,15 +29,15 @@
 GROUP_NAME = 'babel.extractors'
 
 DEFAULT_KEYWORDS = {
```

Comment:
Changing the format of the keywords dict should be documented.

In addition, it's a backward-incompatible change for client apps that pass `keywords` as code, so it should probably support the old format as well as the new one.